source/_integrations/imou_life.markdown

```
@@ -0,0 +1,78 @@
+  - button
+  - select
+  - number
+ha_dhcp: true
+ha_integration_type: integration
+---
+
+The Imou integration uses [Imou Open Platform](https://open.imoulife.com/) APIs to operate Imou camera under the account.
+
+By integrating this component, developers can view real-time previews, control devices, and view camera statuses.
+
+## Prerequisites 
```

Markdownlint (0.35.0)

Expected: 0 or 2; Actual: 1
Trailing spaces

(MD009, no-trailing-spaces)